search/include/pcl/search/impl/brute_force.hpp

```diff
@@ -76,7 +76,7 @@ pcl::search::BruteForce<PointT>::denseKSearch (
   std::vector<Entry> result;
   result.reserve (k);
   std::priority_queue<Entry> queue;
-  if (indices_ != NULL)
+  if (indices_ != nullptr)
```

Comment:
Shared pointers have `operator bool`, so it's possible to write:

```
if (indices_)
  ...
```

I think this looks nicer, what do you think?

Reply:
There are a bunch of similar comparisons below, can we also change them?